selenium.go

```diff
@@ -396,6 +412,8 @@ type WebDriver interface {
 	KeyUp(keys string) error
 	// Screenshot takes a screenshot of the browser window.
 	Screenshot() ([]byte, error)
+	// Print takes PrintArgs and returns a PDF representation of the browser window.
+	Print(args PrintArgs) ([]byte, error)
```

Comment:
Will you please add a new test function in internal/seleniumtest/seleniumtest.go. It will be similar to this test and should be added just below it:

https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/internal/seleniumtest/seleniumtest.go#L1481

The function call should be added to commonTests, after this line:

https://github.com/tebeka/selenium/blob/e9100b7f5ac11727841302026707e3961ba14712/internal/seleniumtest/seleniumtest.go#L169

Reply:
@minusnine done